doc/source/reference/optimize.minimize-cobyqa.rst

```diff
@@ -0,0 +1,8 @@
+.. _optimize.minimize-cobyqa:
+
+minimize(method='COBYQA')
+----------------------------------------
```

Comment:
minor: the `---` line length needs to match the length of the line above to keep Sphinx happy

Reply:
Ok, this is updated in the most recent commit.

Cheers,
Tom.